setup.cfg

```diff
@@ -1,3 +1,78 @@
+[metadata]
+name = photutils
+provides = photutils
+version = 0.7.dev
+author = Photutils Developers
+author_email = photutils.team@gmail.com
+license = BSD 3-Clause License
+license_file = LICENSE.rst
+url = https://github.com/astropy/photutils
+github_project = astropy/photutils
+edit_on_github = False
+description = An Astropy package for source detection and photometry
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+keywords = astronomy, astrophysics, photometry, aperture, psf, source detection, background, segmentation, centroids, isophote, morphology
+classifiers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Cython
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Astronomy
+
+[options]
+python_requires = >=3.5
+packages = find:
+setup_requires =
+    numpy>=1.13
+    cython>=0.28
```

Comment:
Here I think you should use an approach similar to astropy core, to only use cython if we are in dev mode (not for a release):

https://github.com/astropy/astropy/blob/master/setup.py#L24

Reply:
OK, will do, thanks!

Reply:
Done!

Reply:
`is_distutils_display_option` comes from `astropy-helpers`.  Will we need a replacement for this without helpers?